netmiko/f5/f5_tmsh_ssh.py

```diff
@@ -28,12 +30,12 @@ def tmsh_mode(self, delay_factor=1):
         self.clear_buffer()
         return None
 
-    def exit_tmsh(self):
+    def exit_tmsh(self) -> Union[str, List[Any], Dict[str, Any]]:
```

Comment:
Should always return a string (given the context/way send_command is used)...so just do a type check that output is a str and return a str